README.md

```diff
@@ -25,49 +47,48 @@ Since this dataset was released by Yelp for academic purposes, there is a pletho
  
 * **Documentation**: [Yelp dataset attributes and types](https://www.yelp.com/dataset/documentation/main)
     
-* **Files**:
+* **Files**: yelp_academic_dataset_business.json, yelp_academic_dataset_checkin.json, yelp_academic_dataset_review.json, yelp_academic_dataset_tip.json, yelp_academic_dataset_user.json
 
-> yelp_academic_dataset_business.json<br>
-> yelp_academic_dataset_checkin.json<br>
-> yelp_academic_dataset_review.json<br>
-> yelp_academic_dataset_tip.json<br>
-> yelp_academic_dataset_user.json
-    
```

Comment:
IMO the previous format looked less clustered

Reply:
Thanks. Just threw those <br> in there because the markdown was rendering poorly. This can be ignored.